### PR TITLE
KAFKA-4860: Allow spaces in paths on windows

### PR DESCRIPTION
**Pull request by klesta490**
When we install kafka on path with spaces, batch files were failing, this PR is trying to fix this issue.

### CODE REVIEW

File: bin/windows/kafka-consumer-perf-test.bat
Comment (vahidhashemian) [inline bin/windows/kafka-consumer-perf-test.bat]: I don't think this comment is needed here, since it would be meaningless after the PR is merged.

File: bin/windows/kafka-topics.bat
Comment (vahidhashemian) [inline bin/windows/kafka-topics.bat]: Any specific reason for removing this line?

File: bin/windows/kafka-reassign-partitions.bat
Comment (vahidhashemian) [inline bin/windows/kafka-reassign-partitions.bat]: another instance, unintentional?

File: bin/windows/kafka-run-class.bat
Comment (vahidhashemian) [inline bin/windows/kafka-run-class.bat]: here too

File: bin/windows/kafka-server-start.bat
Comment (vahidhashemian) [inline bin/windows/kafka-server-start.bat]: here too

File: bin/windows/kafka-topics.bat
Comment (klesta490) [reply bin/windows/kafka-topics.bat]: no, mistake

File: bin/windows/kafka-reassign-partitions.bat
Comment (klesta490) [reply bin/windows/kafka-reassign-partitions.bat]: mistake

File: bin/windows/kafka-run-class.bat
Comment (klesta490) [reply bin/windows/kafka-run-class.bat]: another one :)

**Comment by vahidhashemian** [comment]
@klesta490 Did you want to submit an update to this PR? Thanks.

**Comment by klesta490** [comment]
yes, i will look o it during weekend

Dne 11. 3. 2017 0:37 napsal uživatel "Vahid Hashemian" <
notifications@github.com>:

**Comment by klesta490** [comment]
Fixed :)

**Review by vahidhashemian — APPROVED** [review]
Thanks for updating the patch. LGTM.

**Comment by klesta490** [comment]
Should I do something? Or it it's not in my hands anymore?

**Comment by vahidhashemian** [comment]
@klesta490 I'm cc'ing a couple of committers to take a look and provide feedback.
cc @ijuma @hachikuji for review.

**Comment by klesta490** [comment]
well, well

**Comment by klesta490** [comment]
It would be nice to atleast close this PR if you are obviously not interrested in it. :-/

**Comment by hachikuji** [comment]
@klesta490 Probably our hesitation is due to the fact that not many of us have a windows system to test this on. That said, if @vahidhashemian has validated the changes, that's probably good enough for me.

**Comment by vahidhashemian** [comment]
@hachikuji I tested this PR with spaces in the Kafka folder path on my Windows machine and did not find any issues. Still a +1 from me.

**Review by hachikuji — APPROVED** [review]
LGTM

**Pull request closed without merging**
